controllers/ovndbcluster_controller.go

```diff
@@ -603,7 +603,10 @@ func (r *OVNDBClusterReconciler) reconcileNormal(ctx context.Context, instance *
 		return ctrlResult, nil
 	}
 
-	instance.Status.ReadyCount = sfset.GetStatefulSet().Status.ReadyReplicas
+	deploy := sfset.GetStatefulSet()
```

Comment:
hmm we shoudn't use "deploy" vars for statefulsets :) , we need to cleanup some conditions too for this.

Reply:
I can do that, I kept it as `deploy` since a statefulset is basically also a deployment. let me change it

Reply:
done